README.md

```diff
@@ -72,27 +72,28 @@ module "castai-eks-cluster" {
 
       custom_taints = [
         {
-          key = "custom-taint-key-1"
+          key   = "custom-taint-key-1"
           value = "custom-taint-value-1"
         },
         {
-          key = "custom-taint-key-2"
+          key   = "custom-taint-key-2"
           value = "custom-taint-value-2"
         }
       ]
 
       constraints = {
         fallback_restore_rate_seconds = 1800
-        spot = true
-        use_spot_fallbacks = true
-        min_cpu = 4
-        max_cpu = 100
-        instance_families = {
+        spot                          = true
+        use_spot_fallbacks            = true
+        min_cpu                       = 4
+        max_cpu                       = 100
+        instance_families             = {
           exclude = ["m5"]
         }
         compute_optimized = false
         storage_optimized = false
-        architectures = ["amd64"]
+        is_gpu_only       = false
```

Comment:
You should also bump the provider version requirements. For now module require >4.1.0, `is_gpu_only` is available from     4.2.0. Module will not work on tf provider 4.1.0 afterh this change 
https://github.com/castai/terraform-provider-castai/releases/tag/v4.2.0 )
@laimonasr